packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -305,6 +313,32 @@ const main = async () => {
         .addOption(
             new program.Option('-w, --wait', 'wait for the deployment to complete before exiting')
         )
+        .addOption(
+            new program.Option(
+                '-u, --user <email>',
```

Comment:
`--user` and `--key` must always be used together. If only one is provided, the command should error. Seems like the easiest way to do this would be a check in the existing `preAction` hook.

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/469255bc6a1869602555973b4eb462b6e31c1fba/packages/pwa-kit-dev/bin/pwa-kit-dev.js#L148-L156